src/pages/TermsOfService.js

```diff
@@ -1,307 +1,34 @@
-
-## Disclaimer of Warranty
-
-You expressly understand and agree that your use of Terra, or any
-material available through it, is at your own risk. Neither the Broad
-nor its employees warrant that Terra will be uninterrupted,
-problem-free, free of omissions, or error-free; nor do they make any
-warranty as to the results that may be obtained from Terra.
-
-## Limitation of Liability
-
-In no event will Broad, its affiliates or participating institutions, or
-their respective directors, officers, employees, faculty members or
-students be liable for any damages, include incidental, indirect,
-special, punitive, exemplary, or consequential damages, arising out of
-your use of or inability to use Terra, including without limitation,
-loss of revenue or anticipated profits, loss of goodwill, loss of data,
-computer failure or malfunction, or any and all other damages.
-
-## HIPAA, Protected Health Information, and the Clinical Compliance Features
-
-Terra is not a Covered Entity as that term is defined in the Health
-Insurance Portability and Accountability Act of 1996, as amended, and
-its related regulations (collectively, "HIPAA"). On occasion, Terra
-may agree in writing with a user to perform services for the user in the
-storing PHI. We recommend that such users enter into a formal agreement
-with Terra/Firecloud.
-
-Terra does offer clinical compliance features as part of its service
-("Compliance Features") for users who wish to upload, store, or
-otherwise transfer PHI, as well as users who are using the Site in
-connection with their clinical operations. Users who desire to upload,
-store, or otherwise transfer PHI using the Site must implement all of
-the required Clinical Features and must enter into a formal agreement
-with Terra/Firecloud stating that. The uploading, storing, or
-transferring of PHI using the Site by users that have not implemented
-the Clinical Features is strictly prohibited. You agree that, unless you
-have implemented the Clinical Features, you will not upload, store, or
-otherwise transfer PHI using the Site.
-
-You acknowledge that this may require you, in some instances, to
-anonymize sequence data prior to uploading it to the Site. You further
-agree to indemnify and hold harmless Terra of and from any and all
-claims, demands, losses, causes of action, damage, lawsuits, judgments,
-including attorneys' fees and costs, arising out of or relating to your
-uploading, storing, or transferring of PHI without having fully
-implemented the Clinical Features.
-
-Terms as of February 12, 2020.
-`
-
 const TermsOfServicePage = () => {
   const [busy, setBusy] = useState()
   const { isSignedIn, acceptedTos } = authStore.get() // can't change while viewing this without causing it to unmount, so doesn't need to subscribe
   const needToAccept = isSignedIn && !acceptedTos
+  const [termsOfService, setTermsOfService] = useState('')
+  const [tosLoaded, setTosLoaded] = useState(false)
```

Comment:
You can get rid of this state entirely if you use `!!termsOfService` as the render condition